config/gregtech/worldgen/vein/nether/sulfur_deposit_vein.json

```diff
@@ -0,0 +1,45 @@
+{
+  "weight": 120,
+  "density": 1,
```

Comment:
I would make this density slightly smaller, maybe like 0.8

Reply:
(this is just to intersperse some netherrack into the deposit to make it look cool as an outcropping)